container-core/src/main/java/com/yahoo/container/handler/VipStatus.java

```diff
@@ -24,20 +24,25 @@ public VipStatus() {
     }
 
     public VipStatus(QrSearchersConfig dispatchers) {
-        this(dispatchers, new VipStatusConfig(new VipStatusConfig.Builder()), new ClustersStatus());
+        this(dispatchers, new ClustersStatus());
     }
 
     public VipStatus(ClustersStatus clustersStatus) {
         this.clustersStatus = clustersStatus;
     }
 
     @Inject
-    public VipStatus(QrSearchersConfig dispatchers, VipStatusConfig vipStatusConfig, ClustersStatus clustersStatus) {
+    public VipStatus(QrSearchersConfig dispatchers, ClustersStatus clustersStatus) {
         this.clustersStatus = clustersStatus;
-        clustersStatus.setReceiveTrafficByDefault(vipStatusConfig.initiallyInRotation());
         clustersStatus.setContainerHasClusters(! dispatchers.searchcluster().isEmpty());
     }
 
+    /** @deprecated don't pass VipStatusConfig */
+    @Deprecated // TODO: Remove on Vespa 8
+    public VipStatus(QrSearchersConfig dispatchers, VipStatusConfig vipStatusConfig, ClustersStatus clustersStatus) {
```

Comment:
Rename vipStatusConfig arg to 'ignored', so it's easily seen when used in the IDE

Reply:
Ok, done.